src/jsifier.js

```diff
@@ -97,6 +97,22 @@ function getTransitiveDeps(symbol) {
   return Array.from(transitiveDeps);
 }
 
+function shouldPreprocess(fileName) {
+  return read(fileName).match(/'use preprocessor';\n/) != null;
```

Comment:
should we match `"` as well as `'`? That works for `use strict` so I think maybe we should.